andela/ah-frontend-realers
# #166187819 User is able to edit their profile

#### What does this PR do?

Enables users to edit their profile

#### Description of task completed
* created actions, reducers and tests relevant for viewing  editing it.

#### How should this be manually tested on your local machine
* Do a `git fetch origin ft-edit-profile-66187819`
* Do a `git checkout ft-edit-profile-66187819`
* Run this command `$ npm install`
* Run this command `$ npm start`
*make sure you sign up and login
* update the url in the browser by adding `/editProfile` for viewing profiles 
*You should be able to view your profile 



[finishes #166187819](https://www.pivotaltracker.com/n/projects/2350340/stories/166187819)
